libcxx/test/benchmarks/containers/flat_map.bench.cpp

```diff
@@ -0,0 +1,25 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: c++03, c++11, c++14, c++17, c++20, c++23
+
+#include <flat_map>
```

Comment:
Since this file is new I wonder, should we add test for the unordered associative containers?

Reply:
I agree, I think these benchmarks also work pretty well for unordered containers, with minor tweaks. Doing.